oauthenticator/tests/test_google.py

```diff
@@ -72,37 +75,55 @@ async def test_admin_google_groups(google_client):
-    whitelist_user_info = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakegroup'])
-    whitelisted_user_groups = whitelist_user_info['auth_state']['google_user']['google_groups']
-    admin_user = whitelist_user_info['admin']
-    assert 'fakegroup' in whitelisted_user_groups
+    handler = google_client.handler_for_user(user_model('fakealloweduser@email.com'))
+    allowed_user_info = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakegroup'])
+    allowed_user_groups = allowed_user_info['auth_state']['google_user']['google_groups']
+    admin_user = allowed_user_info['admin']
+    assert 'fakegroup' in allowed_user_groups
     assert admin_user == False
-    handler = google_client.handler_for_user(user_model('fakenonwhitelisted@email.com'))
-    whitelisted_user_groups = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakenonwhitelistedgroup'])
-    assert whitelisted_user_groups is None
+    handler = google_client.handler_for_user(user_model('fakenonalloweduser@email.com'))
+    allowed_user_groups = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakenonallowedgroup'])
+    assert allowed_user_groups is None
 
 
-async def test_whitelisted_google_groups(google_client):
+async def test_allowed_google_groups(google_client):
     authenticator = GoogleOAuthenticator(
         hosted_domain=['email.com', 'mycollege.edu'],
-        google_group_whitelist={'email.com': ['fakegroup']}
+        allowed_google_groups={'email.com': ['fakegroup']}
     )
     handler = google_client.handler_for_user(user_model('fakeadmin@email.com'))
     admin_user_info = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakeadmingroup'])
     assert admin_user_info is None
-    handler = google_client.handler_for_user(user_model('fakewhitelisted@email.com'))
-    whitelist_user_info = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakegroup'])
-    whitelisted_user_groups = whitelist_user_info['auth_state']['google_user']['google_groups']
-    admin_field = whitelist_user_info.get('admin')
-    assert 'fakegroup' in whitelisted_user_groups
+    handler = google_client.handler_for_user(user_model('fakealloweduser@email.com'))
+    allowed_user_info = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakegroup'])
+    allowed_user_groups = allowed_user_info['auth_state']['google_user']['google_groups']
+    admin_field = allowed_user_info.get('admin')
+    assert 'fakegroup' in allowed_user_groups
     assert admin_field is None
-    handler = google_client.handler_for_user(user_model('fakenonwhitelisted@email.com'))
-    whitelisted_user_groups = await authenticator.authenticate(handler, google_groups=['anotherone', 'fakenonwhitelistedgroup'])
-    assert whitelisted_user_groups is None
+    handler = google_client.handler_for_user(user_model('faallowedisted@email.com'))
```

Comment:
A code suggestion based on mirroring the logic deleted rather than understanding it properly. I assume this string doesn't matter though.

```suggestion
    handler = google_client.handler_for_user(user_model('fakenonalloweduser@email.com'))
```